xml/slurm.xml

```diff
@@ -249,31 +249,31 @@ sbatch sleeper.sh
     dependence of the job which a user has run in the history.
    </para>
    <para>
-    The slurm database has two components which is the daemon
-    <literal>slurmdbd</literal> itself and a mysql database, where
-    <literal>mariab</literal> is recommended. The database can be on the same
+    The slurm database has two components: the <literal>slurmdbd</literal>
+    daemon itself, and a MySQL database, where <literal>mariab</literal> is
+    recommended. The database can be on the same
     node which runs <literal>slurmdbd</literal> or another node. For this
-    simple setup all these services run on the same node.
+    simple setup, all these services run on the same node.
    </para>
    <procedure>
     <title>Install <package>slurmdbd</package></title>
     <note>
-     <title>Mariadb</title>
+     <title>MariaDB</title>
      <para>
       If you want to use an external SQL database (or such a database is
-      already installed on the control node), you can skip the Mariadb-related
+      already installed on the control node), you can skip the MariaDB-related
       steps.
      </para>
     </note>
     <step>
      <para>
-      Install the Mariabd SQL database with <command>zypper in
+      Install the MariaDB SQL database with <command>zypper in
       mariadb</command>.
      </para>
     </step>
     <step>
      <para>
-      Start and enable Mariadb with the following commands:
+      Start and enable MariaDB with the following commands:
```

Comment:
There should be an entity for MariaDB, maybe use that. (Appears multiple times.)